lib/std/os.zig

```diff
@@ -3327,3 +3327,37 @@ pub fn getrusage(who: i32) rusage {
         else => unreachable,
     }
 }
+
+pub const TermiosError = error {
+    NotATerminal,
+    InvalidAction,
+    ProcessOrphaned,
+} || UnexpectedError;
+
+pub fn tcgetattr(handle: fd_t) TermiosError!termios {
```

Comment:
`TermiosError` doesn't seem like the correct error set for `tcgetattr`? `ProcessOrphaned` doesn't appear to be possible?

Reply:
`ProcessOrphaned` is returned when [tcsetattr()](https://pubs.opengroup.org/onlinepubs/000095399/functions/tcsetattr.html) returns `EIO`. I haven't run into this error myself but I'm just following the standard.

Can you elaborate more on why it's not the correct error set?

Reply:
I was pointing out that `get` and `set` should have different error sets.